ngmix/defaults.py

```diff
@@ -8,3 +8,20 @@
 # for priors etc.
 LOWVAL = -np.inf
 BIGVAL = 9999.0e47
+
+DEFAULT_LM_PARS = {"maxfev": 4000, "ftol": 1.0e-5, "xtol": 1.0e-5}
+
+ADMOM_RESULT_DTYPE = [
```

Comment:
I'm not 100% sure this default belongs here.

Reply:
This is a result of the refactoring into sub-packages.

I had to take it out of the package to use it in another package due to circular imports.

Reply:
wasn't a problem when they were plain modules, something special about packages

Reply:
ah right. Fine by me ofc. Usually, a circle import indicates a design issue, but I don't think we should care.

Reply:
could make another module `types.py` or something and put others there as well

Reply:
looking again there is  no reason to re-use this; it is I think OK to just copy it

Reply:
Ahhhh! I think that was the design flaw. They are simply not the same thing. Too DRY!

Reply:
agreed, too dry :)